Assets/Character/Scripts/Constants.cs

```diff
@@ -4,4 +4,22 @@
 	public const int BASE_JUMP_SPEED = 300;
     public const int BASE_ROLL_SPEED = 100;
     public const float ROLL_TIME = 1f;
-}
\ No newline at end of file
+    public const float PLAYER_STAMINA_CHARGE_TIME = .3f;
+
+    public static int GetStaminaCost(PlayerStateEnum state)
+    {
+        switch (state)

```

Comment:
this makes more sense to be on each PlayerState class (state.getStaminaCost()) but either way they will be constants defined in the constants file.

keep in mind that all of these "constants" should be configurable via Unity Editor